File: ngMan/src/ngMan.API/Controllers/ArticleController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Mvc;
using ngMan.Common.Model;

namespace ngMan.API.Controllers
{
    [Route("api/[controller]")]
    public class ArticleController:Controller
    {
        [HttpGet("{articleId}")]
        public Article GetArticle(int articleId)
        {
            return new Article
            {
                Title = "My Firt Article",
                Content = "This is my firt blog article!"
            };
        }
    }
}

File: ngMan/src/ngMan.Web/wwwroot/article/view/ArticleController.ts
/// <reference path="../../boots.ts" />
/// <reference path="../model/article.ts" />

  module ngMan.article.View {
    import Article = ngMan.article.Model.Article;
    class ArticleController {
        public article: Article;
        static $inject =["articleService"];
        constructor(private articleService:ngMan.article.Services.ArticleService) {
            this.article = new Article();
            this.article.title = "My Title";
            this.article.content = "My Content";
        }

        public getArticle() {
            this.articleService.getArticle().then(promiseValue => {
                this.article = promiseValue.data;
            });
        }
    }

    angular.module("article").controller("articleController", ArticleController);
  }